test/quantization/fx/test_quantize_pt2e.py

```diff
@@ -299,6 +314,61 @@ def forward(self, x):
         self.checkGraphModuleNodes(
             m, expected_node_list=node_list, expected_node_occurrence=node_occurrence)
 
+    def test_prepare_qat_conv_bn_fusion(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv = torch.nn.Conv2d(3, 3, 3)
+                self.bn = torch.nn.BatchNorm2d(3)
+
+            def forward(self, x):
+                x = self.conv(x)
+                x = self.bn(x)
+                return x
+
+        import torch.ao.quantization._pt2e.quantizer.qnnpack_quantizer as qq
+        quantizer = QNNPackQuantizer()
+        quantizer.set_global(qq.get_default_per_channel_symmetric_qnnpack_operator_spec(is_qat=True))
+        m = M()
+        example_inputs = (torch.randn(1, 3, 5, 5),)
+
+        # program capture
+        m, guards = torchdynamo.export(
+            m,
+            *copy.deepcopy(example_inputs),
+            aten_graph=True,
+            tracing_mode="real",
+        )
+
+        m = prepare_qat_pt2e_quantizer(m, quantizer)
+        m(*example_inputs)
+
+        # Verify that the subgraph was replaced successfully with fake quantizes inserted
```

Comment:
can we just check the result model graph directly

Reply:
yeah, that seems more robust?

Reply:
Sure. I will just match it manually